src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -190,14 +190,14 @@ public final void refreshItems() {
         // if it actually does on platforms other than Linux...
         listView.setItems(null);
         listView.setItems(getElementsList());
+
         issuesCount = getElementsList().size();
         closedIssuesCount = getClosedIssuesCount();
         openIssuesCount = issuesCount - closedIssuesCount;
-        listView.restoreSelection();
-        this.setId(IdGenerator.getPanelId(panelIndex));
         updateFooter();
 
-
+        listView.restoreSelection();
```

Comment:
Does the relative order of this and `updateFooter` matter? Feels like it shouldn't.